docs/workspace/catalog.md

```diff
@@ -4,13 +4,42 @@ sidebar_position: 4
 
 # Catalog
 
-The **Data Catalog** is a comprehensive listing of all [geospatial datasets](../further_reading/glossary#geospatial-data) available for exploration and analysis. A wide range of datasets are provided by Plan4Better, our authoritative partners and [open source](../further_reading/glossary#open-source) libraries, ensuring a **reliabile and accurate** database for your GIS projects.
+The **Data Catalog** is a comprehensive listing of all [geospatial datasets](../further_reading/glossary#geospatial-data) available for exploration and analysis. A wide range of datasets are provided by Plan4Better, our authoritative partners, and [open source](../further_reading/glossary#open-source) libraries, ensuring a **reliable and accurate** database for your GIS projects.
```

Comment:
I don't think "authoritative partners" is an appropriate phrase here. Can we change this to just "... Plan4Better, our partners, and ..."